src/Scheduler.php

```diff
@@ -43,6 +45,15 @@ protected function _scheduleStaticJobs(): SchedulerInterface
         $schedulerCache = $this->_getSchedulerCache();
         $this->_getSchedulerJobCollection()->setReferenceDateTime($this->_getTime()->getNow());
         foreach ($this->_getSchedulerJobTypeCollection()->getIterator() as $jobType) {
+            if (!$jobType->getIsEnabled()) {
+                $this->_getLogger()->warning(
+                    sprintf(
+                        'Scheduler skipping disabled job type [%s]',
+                        $jobType->getCode()
+                    )
```

Comment:
```suggestion
                    , ['disabled_job_skipped' => $jobType->getCode()] )
```
Can we create a machine readable context object along the following lines?

Reply:
Anything for you, buddy